### ISSUE TITLE
It should be possible to have access to an "execution tag"

### ISSUE THREAD

**Issue by balves** [robotframework/robotframework#1979]
I don't know if it is possible, but from what i have been reading apparently it is not.
It should be possible to have access to the value of the tag currently in execution. 
When on the command line you pass --include <tag> you should be able to access that value on the test case.
Could be handy if launching pybot within a for loop and with different tag values, to access that tag value and decide in runtime how the test will behave depending on the tag.

Best Regards

**Comment by laurentbristiel** [comment]
I don't think there is *one* tag currently in execution. Tags are more a filter that select a set of tests to be run, but a single test could match several tags (that could all be included)... so there would not be a "current tag".

**Comment by pekkaklarck** [comment]
Laurent is right that there is no single tag. Instead there are lists of include and exclude patterns. I guess we could expose them, but I'm afraid using them would still be a bit tricky. Personally I've used variables when I've needed to pass information like this to tests from the command line. This example has a little duplication, but perhaps it would anyway work for you:

    pybot --include smoke --variable execute:smoke tests.robot
